templates/server-ingress.yaml

```diff
@@ -8,11 +8,7 @@
 {{- $serviceName = printf "%s-%s" $serviceName "active" -}}
 {{- end }}
 {{- $servicePort := .Values.server.service.port -}}
-{{ if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
-apiVersion: networking.k8s.io/v1beta1
-{{ else }}
-apiVersion: extensions/v1beta1
-{{ end }}
+apiVersion: networking.k8s.io/v1
```

Comment:
Since networking.k8s.io/v1 Ingress has only been available since k8s 1.19, I think we'll need to maintain some of the older apiVersion's here, back to k8s 1.14 if possible.

Reply:
done